Yoast/wordpress-seo
# Unify all upsell buttons in yellow design

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:
* I introduced a new `yoast-button-upsell` class to style buttons that had WP's `primary-button` class before.
* I've removed the `yoast-button-go-to` class of some buttons, because that class added » at the end of buttons. 
* In admin-global.scss I use a line height of 42px to vertically align the text in the button. It works, but this 42 is a magic number. Also, as a result, the button looks strange on smaller screens when the text goes to multiple lines. A solution is very welcome:
<img width="301" alt="schermafbeelding 2018-10-05 om 11 52 09" src="https://user-images.githubusercontent.com/17744553/46528861-1f8fcc00-c895-11e8-87c0-e8206368658c.png">

## Test instructions

This PR can be tested by following these steps:

* Test all buttons mentioned in the issue. The design is based on the [buttons on yoast.com (scroll down a bit)](https://yoast.com/readability-ranks/).
* Check if no button text ends with 'now!' anymore
* Test with different viewport sizes.
* Don't forget to test hover styles, focus styles and active styles.
* If possible, test in different browsers.

## Quality assurance


Fixes #10980
